src/command_modules/azure-cli-cloud/azure/cli/command_modules/cloud/_params.py

```diff
@@ -21,14 +21,40 @@ def get_custom_cloud_name_completion_list(prefix, action, parsed_args, **kwargs)
                       completer=get_cloud_name_completion_list)
 
 register_cli_argument('cloud show', 'cloud_name',
-                      help='Name of the registered cloud. If not specified, we use the current active cloud.')
+                      help='Name of the registered cloud. If not specified, we use the active cloud.')
```

Comment:
Can we not simply have the default value get the active cloud name (like VM admin username)? Then we don't need this "If not specified..." text and it will be clear what the active cloud is.